subprojects/docs/src/docs/release/notes.md

```diff
@@ -78,6 +78,14 @@ Depending on the version of Java you use, Gradle will negotiate TLS v1.2 or TLS
 which causes the exception `javax.net.ssl.SSLException: No PSK available. Unable to resume`. If you run into this issue,
 we recommend updating to the latest minor JDK version.
 
+## IDE integration
+
+### Importing projects with custom source sets 
```

Comment:
How does it affect IDEA and Android Studio?

Reply:
This is a change specific to Eclipse. I'll adjust the tile.